src/partition/ReceivedPartition.cpp

```diff
@@ -238,14 +238,18 @@ void ReceivedPartition::compute()
 }
 
 namespace {
-std::string errorMeshFilteredOut(const std::string &meshName)
+std::string errorMeshFilteredOut(const std::string &meshName, const int rank)
```

Comment:
As you already touch this function, could you please port it to fmtlib?

This also allows you to remove all the `+` from the message and use consecutive string literals instead.

Please use `utils/fmt.hpp`.
```
#include <utils/fmt.hpp>
return fmt::format("...", meshName, rank);
```

Reply:
Done as well